src/lib_ccx/ccx_common_constants.h

```diff
@@ -117,42 +117,42 @@ enum ccx_stream_type
 	CCX_STREAM_TYPE_ISO_IEC_13818_6_TYPE_B = 0x0B,
 	CCX_STREAM_TYPE_ISO_IEC_13818_6_TYPE_C = 0x0C,
 	CCX_STREAM_TYPE_ISO_IEC_13818_6_TYPE_D = 0x0D,
-	CCX_STREAM_TYPE_AUDIO_AAC   = 0x0f,
-	CCX_STREAM_TYPE_VIDEO_MPEG4 = 0x10,
-	CCX_STREAM_TYPE_VIDEO_H264  = 0x1b,
-	CCX_STREAM_TYPE_PRIVATE_USER_MPEG2=0x80,
+	CCX_STREAM_TYPE_AUDIO_AAC   = 0x0f, // Should there really be 0x0f?
+	CCX_STREAM_TYPE_VIDEO_MPEG4 = 0x10, // Should there really be 0x10?
+	CCX_STREAM_TYPE_VIDEO_H264  = 0x1b, // Should there really be 0x1b?
+	CCX_STREAM_TYPE_PRIVATE_USER_MPEG2 = 0x80,
 	CCX_STREAM_TYPE_AUDIO_AC3   = 0x81,
 	CCX_STREAM_TYPE_AUDIO_HDMV_DTS = 0x82,
-	CCX_STREAM_TYPE_AUDIO_DTS   = 0x8a,
+	CCX_STREAM_TYPE_AUDIO_DTS   = 0x8a // Should there really be 0x8a?
 };
 
 enum ccx_mpeg_descriptor
 {
-	CCX_MPEG_DSC_REGISTRATION = 0x05,
+	CCX_MPEG_DSC_REGISTRATION = 0x05, // And what's with all these random numbers?
```

Comment:
They are not random. The enum is called "mpeg descriptor", what else do you want to know where they come from?
If you google for example "mpeg descriptors table teletext" you will find that teletext specifically comes from EN 300 468, which is a public standard.